asteroid/models/demask.py

```diff
@@ -178,3 +160,31 @@ def get_model_args(self):
         }
         model_args.update(self.fb_kwargs)
         return model_args
+
```

Comment:
If we're refactoring we should probably use `self.encoder.get_config()` in `get_model_args`. 
If we make a `DeMaskMasker` module (bad name), with `get_config`, we don't need to write `get_model_args`. This is related to the next comment.

Reply:
Actually we do because `get_model_args` of the base class is missing the arguments `fb_kwargs`, `fb_type`, `input_type` and `output_type`. If we break backward compatibility, we can rename `fb_type` in `fb_name` and maybe remove `fb_kwargs` but have to keep the other arguments right?
Also, the `get_model_args` of the base class adds `encoder_activation`, `fb_name`, `in_chan` and `out_chan` arguments.

I just created a (very) simple class for the masker of `DeMask` but, unlike all the other maskers, it has to inherit from `nn.Sequential` if we want to be backward compatible. Otherwise, the state dict cannot load. Shall it be moved to `asteroid.masknn`?

Just waiting for comments before adding docstrings and type hinting.

Reply:
About `DeMask`'s masker. Sorry for being hesitant, and thanks for being so adapting. 
We'll have `build_demask_masker` as you write earlier, but as a public method. Inhereting from `Sequential` is a trick for BC, and I don't like it.  

You're (mostly) right about `get_model_args`. Why _mostly_?
- `in_chan` and `out_chan` are not added by `get_model_args`
- Additional kwargs passed to the filterbanks are handled in `Filterbank.get_config()` which makes me see that the current code is wrong wrt to `fb_kwargs` . 

```python
from asteroid import DeMask

model = DeMask(fb_type="param_sinc", min_low_hz=60)
In [53]: model.get_model_args()
Out[53]: 
{'input_type': 'mag',
 'output_type': 'mag',
 'hidden_dims': [1024],
 'dropout': 0.0,
 'activation': 'relu',
 'mask_act': 'relu',
 'norm_type': 'gLN',
 'fb_type': 'param_sinc',
 'n_filters': 512,
 'stride': 256,
 'kernel_size': 512,
 'fb_kwargs': {'min_low_hz': 60},
 'sample_rate': 16000,
 'min_low_hz': 60}
```

Again, thanks for the flexibility. I'll take it from here.